entrypoint

```diff
@@ -17,6 +17,11 @@ if [[ $# -eq 0 ]]; then
         the source of the \`nextstrain\` command for examples.
     "
 else
-    # Otherwise, exec into whatever command is provided.
-    exec "$@"
+    # Otherwise, exec into whatever command is provided…
+    if [[ "$(id -u):$(id -g)" == 0:0 ]]; then
```

Comment:
[TIL](https://superuser.com/questions/626843/does-the-root-account-always-have-uid-gid-0) "root" is just a name but uid will always be 0.

Reply:
Yep, similar to how PID 1 is special.